src/main/java/Task.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Scanner;
+public class Task {
+    private String taskName;
+    private boolean done;
```

Comment:
Perhaps naming the boolean more appropriately would be better!
```suggestion
    private boolean isDone;
```